spec/elastic_apm/system_info_spec.rb

```diff
@@ -12,6 +12,12 @@ module ElasticAPM
           expect(subject[key]).to_not be_nil
         end
       end
+
+      context 'hostname' do
+        it 'has no newline at the end' do
+          expect(subject[:hostname]).to eq(`hostname`.chomp)
```

Comment:
Makes it a bit weird to have the same approach both places. Maybe `.to_not match(\n\z)` ?

Reply:
Yeah you’re right. Will push another one.